common/lib/protocol-definitions/package.json

```diff
@@ -71,6 +71,13 @@
 		"typescript": "~5.1.6"
 	},
 	"packageManager": "pnpm@7.32.3+sha512.c28d73f0d82a6c4712a40b91aa43d7397e119b16f35e5e75fe9b7e8fd4f2fc6dfbb68bb3ffac3473a958bbafa7888b79ec384ad122537378089a3a2a19b9900e",
+	"fluidBuild": {
+		"tasks": {
+			"tsc": [
+				"typetests:gen"
+			]
+		}
+	},
```

Comment:
We need this one because there's no separate test build for this package, so we want type tests to be compiled (i.e. validated) with the rest of the source?

Reply:
I was under the impression that generating typetests required a compilation pass on the src first but can't recall exactly why... otherwise why we don't do this by default?